tests/conftest.py

```diff
@@ -55,6 +57,12 @@ def eager_task_loop_factory() -> asyncio.AbstractEventLoop:
 
 @pytest.fixture(autouse=True)
 def blockbuster() -> Iterator[BlockBuster]:
```

Comment:
Not sure why mypy didn't catch this.
```suggestion
def blockbuster() -> Iterator[BlockBuster | None]:
```